### PR TITLE
update generator automated_detection_testing

### PR DESCRIPTION
**Pull request by patel-bhavin**
Tested and update docker file on AWS.

### CODE REVIEW

File: automated_detection_testing/detection_service.py [hunk 1]
```diff
@@ -79,13 +80,14 @@ def main(args):
...
+    private_key_path = str(os.getcwd() + "/" + ssh_key_name)
 
     # build new version of ESCU
     sys.path.append(os.path.join(os.getcwd(),'security_content/bin'))
+
     try:
         module = __import__('generate')
-        module.sys.argv = ['generate', '-p', 'security_content', '-o' 'security_content/package']
-        results = module.main(module.sys.argv)
+        results = module.main(REPO_PATH = 'security_content' , OUTPUT_PATH = 'security_content/package', VERBOSE = '--verbose' )
```
Comment (P4T12ICK) [inline automated_detection_testing/detection_service.py]: @patel-bhavin, I think the correct value for VERBOSe is either True or False. Maybe we should go for TRUE.

**Review by P4T12ICK — CHANGES_REQUESTED** [review]

File: automated_detection_testing/detection_service.py (hunk 1 above)
Comment (patel-bhavin) [reply automated_detection_testing/detection_service.py]: Updated! Thanks for reviewing this

Comment (P4T12ICK) [reply automated_detection_testing/detection_service.py]: Another small thing: The True needs to be without ' otherwise it's a string and not a boolean.

Comment (patel-bhavin) [reply automated_detection_testing/detection_service.py]: hmmn, weird. it still worked for me and gives a verbose output. anyway, i updated it to boolen

**Review by P4T12ICK — APPROVED** [review]

**Pull request merged**